dockerfiles/agent/Dockerfile

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+
+FROM alpine:3.11
+
+ENV USER=user
+ENV UID=12345
+ENV GID=23456
```

Comment:
saw it in https://stackoverflow.com/questions/49955097/how-do-i-add-a-user-when-im-using-alpine-as-a-base-image/55757473#55757473 hopefully we will not have any surprises on OpenShift ;)